extensions.yaml

```diff
@@ -28,5 +28,12 @@ extensions:
  #datalad_metalad:
  # pypi: datalad-metalad
  datalad_crawler:
- # datalad_webapp,     # not developed ATM, not sure if of interest to keep track
+ datalad_webapp:
+  pypi: datalad-webapp
+  package: datalad_webapp
+ datalad_hirni:
+  apt_depends: [dcm2niix]
+  pypi: datalad-hirni
+  package: datalad_hirni
+  github: /psychoinformatics-de/datalad-hirni
```

Comment:
FWIW, I would have kept leading `/` out of this one here, and explicit in the urls like `https://github.com/{e. ...`

Reply:
Well, can do it that way, too. Leading `/` is just a slightly simpler thing to check for. Doesn't really matter to me.